Makefile

```diff
@@ -1,10 +1,15 @@
+LD_FLAGS=-ldflags " \
+    -X github.com/cnoe-io/idpbuilder/pkg/cmd/version.idpbuilderVersion=$(shell git describe --always --tags --dirty --broken) \
```

Comment:
`--always` flag means that this won't fail with forks/local dev and not output anything for `idpbuilderVersion` if there are no local tags.